DataXResearch/rabbithole
# UI overhaul

Closes #17, #25, #14, #16 

## Changes
- clean up timeline, sidebar
- fix logo
- rename project
- delete project
- delete website from project
- synchronize project details when switching tabs
- some other bug fixes (in commit history)

## Test
- Existing functionality (create project, add website to project)
- Delete project 
- Delete website from project
- Rename project
- Ensure sidebar buttons trigger refresh (except sync)
